src/neptune/attributes/namespace.py

```diff
@@ -91,14 +92,14 @@ def to_dict(self) -> Dict[str, Any]:
                 result[key] = value
         return result
 
-    def assign(self, value: Union[NamespaceVal, dict, Mapping], wait: bool = False):
+    def assign(self, value: Union[NamespaceVal, dict, Mapping], *, wait: bool = False):
```

Comment:
Just out of curiosity - do we need dict, when we have Mapping?

Reply:
IDK